viewport-segments-api/index.css

```diff
@@ -0,0 +1,194 @@
+/* Document styles */
+
+html {
+  font-family: helvetica, arial, sans-serif;
+  box-sizing: border-box;
+  height: 100%;
+}
+
+body {
+  margin: 0;
+  height: 100%;
+  background: #ffc8dd;
+}
+
+.wrapper {
+  height: 100%;
+  display: flex;
+}
+
+.fold {
+  background-color: #999;
+}
+
+/* Layout for the two main panes */
+
+.wrapper,
+.list-view,
+.detail-view {
+  position: relative;
+}
+
+.list-view,
+.detail-view {
+  margin: 20px;
+  background: white;
+}
+
+.list-view {
+  overflow: auto;
+  padding: 0 20px;
+  border-bottom: 20px solid white;
+  border-top: 20px solid white;
+}
+
+.detail-view {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 20px;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+ul li {
+  margin-bottom: 20px;
+  list-style-type: none;
+}
+
+a {
+  color: black;
+  display: block;
+  padding: 20px;
+  outline: none;
+  text-decoration: none;
+  border: 1px solid rgb(0 0 0 / 0.3);
+  transition: 0.4s all;
+}
+
+a:hover,
+a:focus {
+  border-left: 20px solid rgb(0 0 0 / 0.3);
+}
+
+ul li:nth-child(odd) {
+  background: #cdb4db;
+}
+
+ul li:nth-child(even) {
+  background: #ffafcc;
+}
+
+/* Heading and paragraph styles */
+
+h1 {
+  margin: 0 0 20px;
+}
+
+.posture-output {
+  background: #cdb4db;
+  padding: 20px;
+  margin: 0;
+  border: 1px solid rgb(0 0 0 / 0.3);
+}
+
+.segment-output {
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  background: #663880;
+  color: white;
+  padding: 10px;
+  border: 1px solid rgb(0 0 0 / 0.3);
+}
+
+.segment-output h2 {
+  margin: 0;
+  font-size: 1.4rem;
+}
+
+.segment-output p {
+  margin: 10px 0 0 0;
+}
+
+/* Media queries */
+
+@media (horizontal-viewport-segments: 1) and (orientation: landscape) {
```

Comment:
I don't think you should mix this with the orientation. If there is a single segment it's just the regular normal case, just use the orientation property.